worlds/ufo50/games/barbuta/locations.py

```diff
@@ -36,19 +37,21 @@ class LocationInfo(NamedTuple):
     "Chest - H5": LocationInfo(21, "Starting Area"),  # $100, be fast before the ledge breaks
     "Chest - H7": LocationInfo(22, "G7 and Nearby"),  # $50
     "Wand Trade - H7": LocationInfo(23, "Wand Trade Room"),  # probably should just have it give you the check
+
+    "Garden": LocationInfo(100, "Boss Area"),
+    "Gold": LocationInfo(101, "Menu"),
+    "Cherry": LocationInfo(102, "Boss Area")
 }
 
 
-def get_locations(base_id: int) -> Dict[str, int]:
+def get_locations() -> Dict[str, int]:
+    base_id = game_ids["Barbuta"]
```

Comment:
Isn't this base_id going to be 40?
It should be 40*1000 + BASE_ID, so that they don't overlap and all that.

Reply:
Woops, good catch